UNRELEASED.md

```diff
@@ -6,6 +6,8 @@
 
 ### Bug fixes
 
+- `Filters` only show the "More filters" button if necessary ([#](https://github.com/Shopify/polaris-react/pull/2856)).
```

Comment:
Let’s move this to "Enhancements" and update the copy a bit (also missing the PR number):

```suggestion
- Updated `Filters` to only show the "More filters" button if necessary ([#2856](https://github.com/Shopify/polaris-react/pull/2856)).
```